pkg/spec/gitops.go

```diff
@@ -1083,65 +1093,72 @@ func parseSoftware(top map[string]json.RawMessage, result *GitOps, baseDir strin
-		if item.Slug == "" {
+	for _, maintainedAppSpec := range software.FleetMaintainedApps {
+		maintainedAppSpec := maintainedAppSpec
+		if maintainedAppSpec.Slug == "" {
 			multiError = multierror.Append(multiError, errors.New("fleet maintained app slug is required"))
 			continue
 		}
 
-		if len(item.LabelsExcludeAny) > 0 && len(item.LabelsIncludeAny) > 0 {
-			multiError = multierror.Append(multiError, fmt.Errorf(`only one of "labels_exclude_any" or "labels_include_any" can be specified for fleet maintained app %q`, item.Slug))
+		if len(maintainedAppSpec.LabelsExcludeAny) > 0 && len(maintainedAppSpec.LabelsIncludeAny) > 0 {
+			multiError = multierror.Append(multiError, fmt.Errorf(`only one of "labels_exclude_any" or "labels_include_any" can be specified for fleet maintained app %q`, maintainedAppSpec.Slug))
 			continue
 		}
 
-		item = item.ResolveSoftwarePackagePaths(baseDir)
+		maintainedAppSpec = maintainedAppSpec.ResolveSoftwarePackagePaths(baseDir)
 
 		// handle secrets
-		if item.InstallScript.Path != "" {
-			if err := gatherFileSecrets(result, item.InstallScript.Path); err != nil {
+		if maintainedAppSpec.InstallScript.Path != "" {
+			if err := gatherFileSecrets(result, maintainedAppSpec.InstallScript.Path); err != nil {
 				multiError = multierror.Append(multiError, err)
 				continue
 			}
 		}
-		if item.PostInstallScript.Path != "" {
-			if err := gatherFileSecrets(result, item.PostInstallScript.Path); err != nil {
+		if maintainedAppSpec.PostInstallScript.Path != "" {
+			if err := gatherFileSecrets(result, maintainedAppSpec.PostInstallScript.Path); err != nil {
 				multiError = multierror.Append(multiError, err)
 				continue
 			}
 		}
-		if item.UninstallScript.Path != "" {
-			if err := gatherFileSecrets(result, item.UninstallScript.Path); err != nil {
+		if maintainedAppSpec.UninstallScript.Path != "" {
+			if err := gatherFileSecrets(result, maintainedAppSpec.UninstallScript.Path); err != nil {
 				multiError = multierror.Append(multiError, err)
 				continue
 			}
 		}
 
-		result.Software.FleetMaintainedApps = append(result.Software.FleetMaintainedApps, &item)
+		result.Software.FleetMaintainedApps = append(result.Software.FleetMaintainedApps, &maintainedAppSpec)
 	}
-	for _, item := range software.Packages {
+	for _, teamLevelPackage := range software.Packages {
 		var softwarePackageSpec fleet.SoftwarePackageSpec
-		if item.Path != nil {
-			softwarePackageSpec.ReferencedYamlPath = resolveApplyRelativePath(baseDir, *item.Path)
+		if teamLevelPackage.Path != nil {
+			softwarePackageSpec.ReferencedYamlPath = resolveApplyRelativePath(baseDir, *teamLevelPackage.Path)
 			fileBytes, err := os.ReadFile(softwarePackageSpec.ReferencedYamlPath)
 			if err != nil {
-				multiError = multierror.Append(multiError, fmt.Errorf("failed to read software package file %s: %v", *item.Path, err))
+				multiError = multierror.Append(multiError, fmt.Errorf("failed to read software package file %s: %v", *teamLevelPackage.Path, err))
```

Comment:
This isn't part of your changes, but since we're here..
```suggestion
				multiError = multierror.Append(multiError, fmt.Errorf("failed to read software package file %s: %w", *teamLevelPackage.Path, err))
```